Extensions/Clipboard.json

```diff
@@ -1,34 +1,111 @@
+      ],
+      "objectGroups": []
+    },
     {
       "description": "Read the text from the clipboard (Windows, macOS, Linux only)",
       "fullName": "Get text from the clipboard (Windows, macOS, Linux)",
       "functionType": "StringExpression",
       "name": "ReadText",
       "sentence": "",
       "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Comment",
+          "color": {
+            "b": 109,
+            "g": 230,
+            "r": 255,
+            "textB": 0,
+            "textG": 0,
+            "textR": 0
+          },
+          "comment": "This is here for retrocompatibility. Try to use the asynchronous readTextCrossPlatform action instead.",
+          "comment2": ""
+        },
         {
           "disabled": false,
           "folded": false,
           "type": "BuiltinCommonInstructions::JsCode",
-          "inlineCode": "eventsFunctionContext.returnValue = \"\";\nif (typeof require === 'undefined') {\n    return;\n} \n\nconst electron = require('electron');\nif (!electron || !electron.clipboard) {\n    return;\n}\n\neventsFunctionContext.returnValue = electron.clipboard.readText();",
+          "inlineCode": "const electron = runtimeScene.getGame().getRenderer().getElectron();\nif (electron && electron.clipboard) eventsFunctionContext.returnValue = electron.clipboard.readText();\n",
           "parameterObjects": "",
           "useStrict": true,
           "eventsSheetExpanded": false
         }
       ],
       "parameters": [],
       "objectGroups": []
+    },
+    {
+      "description": "Write the text in the clipboard (Windows, macOS, Linux only)",
+      "fullName": "Write text to the clipboard (Windows, macOS, Linux)",
+      "functionType": "Action",
+      "name": "WriteText",
+      "sentence": "Write _PARAM1_ to clipboard",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "const electron = runtimeScene.getGame().getRenderer().getElectron();\nconst text = eventsFunctionContext.getArgument(\"text\");\n\nif (electron !== null && electron.clipboard)\n  electron.clipboard.writeText(eventsFunctionContext.getArgument(\"text\"));\nif (\n  typeof navigator !== \"undefined\" &&\n  navigator.clipboard &&\n  navigator.clipboard.writeText\n) navigator.clipboard.writeText(text)\n  .catch(e => console.error(\"Error while writing clipboard: \", e));\nelse {\n  // TODO cache this element to not create elements on each call\n  const tempElement = document.createElement(\"textarea\");\n  tempElement.id = \"Clipboard_copy_temporary\";\n  tempElement.value = text;\n  tempElement.setAttribute('readonly', '');\n  tempElement.position = 'absolute';\n  tempElement.style.left = '-9999px';\n  document.body.appendChild(tempElement);\n  tempElement.select();\n  document.execCommand(\"copy\");\n  tempElement.remove();\n}; \n",
```

Comment:
Is `document.execCommand` useful for a particular browser? Seems `navigator.clipboard` is supported by most browsers (https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText) - apart from IE but we're not compatible with IE anymore.
If we can, let's not have the `document.execCommand` branch at all - simpler and better.

Reply:
For me on google chrome, pasting requires execCommand, even though it said [here](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/readText) that `navigator.clipboard` works on my chrome version. I'd therefore keep it just in case.

Reply:
It's supposed to work since Chrome 66. Can you double check? No way this thing is not working. You might have to accept a permission, which is something we won't be able to avoid (and that must be explained too in the action).

Reply:
Nevermind, there was two factors that caused me to believe this:
1. The extension had a typo in `navigator`🤦‍♂️ 
2. For some security reason, on preview over wifi (which I use for testing in browser with hot reload), `navigator.clipboard` is undefined. As soon as I exported it and hosted it locally though, it worked.